Plugin/Customer/Model/CustomerData.php

```diff
@@ -33,14 +33,12 @@ public function __construct(
      */
     public function afterGetSectionData(Customer $subject, $result)
```

Comment:
why is $subject part of the method if it is not used?

Reply:
magento standard to define customer on this overriding method https://magento.stackexchange.com/questions/134884/how-to-override-getsectiondata-method-of-magento-checkout-customerdata-cart-ph